support-frontend/assets/components/checkoutBenefits/benefitsCheckList.tsx

```diff
@@ -57,18 +58,19 @@ const tableCss = (style: CheckListStyle) => css`
 	}
 `;
 
-export type CheckListData = {
+export type BenefitsCheckListData = {
 	isChecked: boolean;
 	text?: JSX.Element | string;
 	maybeGreyedOut?: SerializedStyles;
 	toolTip?: string;
 	strong?: boolean;
+	isNew?: boolean;
 };
 
 type CheckListStyle = 'standard' | 'compact' | 'hidden';
 
-export type CheckListProps = {
-	checkListData: CheckListData[];
+export type BenefitsCheckListProps = {
+	checkListData: BenefitsCheckListData[];
```

Comment:
It's a little one, but having the prop and the type with different names can be a bit confusing.